src/test/java/org/jitsi/jicofo/bridge/BridgeSelectionStrategyTest.java

```diff
@@ -191,17 +190,11 @@ public void preferRegionWhenStressLevelIsEqual()
             highStressRegion = region1;
 
         highStressBridge.setStats(
-            createJvbStats(StressLevels.HIGH, 100, highStressRegion));
-        assertEquals(
-            highStressBridge.getStressLevel(), StressLevels.HIGH, precision);
+            createJvbStats(750_000, 75_000, highStressRegion));
```

Comment:
What are these numbers?

Reply:
Same below

Reply:
I've added some comments. Please let me know if it doesn't make sense.